src/GafferSceneUI/SceneView.cpp

```diff
@@ -1822,13 +1827,16 @@ void SceneView::contextChanged( const IECore::InternedString &name )
 
 Imath::Box3f SceneView::framingBound() const
 {
-	Imath::Box3f b = m_sceneGadget->selectionBound();
+	PathMatcherDataPtr omitted = new PathMatcherData();
+	omitted->writable().addPath( m_camera->lookThroughCameraPath() );
```

Comment:
When `lookThroughCameraPath()` is empty, I think this is equivalent to adding the root path. We'll need to avoid that if we do decide to check for ancestor matches in `queryBound()`.